hypervisor/common/io_request.c

```diff
@@ -18,11 +18,13 @@ static void fire_vhm_interrupt(void)
 	struct acrn_vm *sos_vm;
 	struct acrn_vcpu *vcpu;
 
-	sos_vm = get_vm_from_vmid(0U);
+	sos_vm = get_sos_vm();
 
-	vcpu = vcpu_from_vid(sos_vm, 0U);
+	if (sos_vm != NULL) {
+		vcpu = vcpu_from_vid(sos_vm, 0U);
```

Comment:
0U or BOOT_CPU_ID?

Reply:
As if the MISRAC check is passed, I am fine. This is minor one

Reply:
oops, I made a mistake, here BOOT_CPU_ID is vcpu id.